lib/real_world/feed/listeners/tracker.rb

```diff
@@ -9,18 +10,19 @@ class Tracker < Colmena::Listener
         def call(event)
           case event.fetch(:type)
           when :article_created
-            followers = port(:router).query(:list_followers).call(
+            RealWorld::Query.iterate_paginated(
+              ->(params) { port(:router).query(:list_followers).call(params) },
               followed_id: event.dig(:data, :author_id),
-            ).fetch(:data)
-
-            followers.each do |follower_id|
-              port(:repository).create(
-                user_id: follower_id,
-                article_id: event.dig(:data, :id),
-              )
+              limit: 1000,
+            ) do |followers|
+              followers.each do |follower_id|
+                port(:repository).create(
+                  user_id: follower_id,
+                  article_id: event.dig(:data, :id),
+                )
```

Comment:
This could crash if the article author gains some followers while looping. Maybe we could check if the feed entry already exists before `create`?